pkg/operator/ceph/nfs/config.go

```diff
@@ -55,5 +55,19 @@ RADOS_KV {
 	pool = "` + spec.ClientRecovery.Pool + `";
 	namespace = "` + spec.ClientRecovery.Namespace + `";
 }
+
+RADOS_URLS {
+	ceph_conf = '` + cephConfigPath + `';
+	userid = ` + userID + `;
+}
+
+%url	rados://` + spec.ClientRecovery.Pool
+
+	if spec.ClientRecovery.Namespace != "" {
+		conf += `/` + spec.ClientRecovery.Namespace
+	}
+
```

Comment:
Note that we're using ClientRecovery.Pool and Namespace here even though the rados configs don't really have anything to do with ClientRecovery.  Maybe we should just do s/ClientRecovery/RADOS/ or something?

Reply:
what would you recommend for the crd schema? Then this object can just follow the naming in the crd. Would `config` be appropriate? Or is there something more descriptive that we're using the rados object for?